diff --git a/renderer.ts b/renderer.ts
index 7d2240b97ff29edb8d8d3b6f99c90f319ea16b2c..ba4b4a688ebe6cc3e9048c3c79a6b37c06252b46 100644
--- a/renderer.ts
+++ b/renderer.ts
@@ -4,6 +4,12 @@ import { defineEventHandler } from 'h3'
 import { defineNitroPlugin } from 'nitropack/dist/runtime/internal/plugin'
 
 export default defineNitroPlugin((nitro) => {
-  const handler = defineEventHandler(() => readAsset('/index.html'))
+  const handler = defineEventHandler(async (event) => {
+    const asset = await readAsset('/index.html')
+    console.log('originalUrl:', event.node.req.originalUrl)
+    console.log('url:', event.node.req.url)
+    console.log('asset:', asset)
+    return asset
+  })
   nitro.router.use('/**', handler)
 })
